### PR TITLE
[TASK] Add copyright notice for Ondřej Mirtes

### LINKED ISSUE #6: License violation
Hello,
this package is clearly and heavily inspired by `phpstan/extension-installer`: https://github.com/phpstan/extension-installer

But there's no mention of the original idea in README or LICENSE file.

`phpstan/extension-installer` is distributed under MIT license (https://github.com/phpstan/extension-installer/blob/master/LICENSE). While the MIT license is very permissive (see the wording) and allows you to copy the code without limitation, the original copyright notice should be preserved.

Please fix this so that the open-source spirit is preserved and we can continue making great work! Thanks.

### PR DESCRIPTION
**Pull request by sabbelasichon**
Resolves: #6

### CODE REVIEW

**Comment by sabbelasichon** [comment]
@TomasVotruba Could you explain why ci stucks at the tests?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+This package is heavily inspired by [phpstan/extension-installer](https://github.com/phpstan/extension-installer) by Ondřej Mirtes. Thank you.
+
 # Rector Extension Installer
 [![Build](https://github.com/rectorphp/rector-installer/workflows/CI/badge.svg)](https://github.com/rectorphp/rector-installer/actions)
```
Comment (TomasVotruba) [inline README.md]: Could you move this to the end of the file? That's usual place for credits.

**Comment by sabbelasichon** [comment]
@TomasVotruba The tests are still stucked. Any ideas what could be the reason for it?

**Comment by TomasVotruba** [comment]
Let's ship it, thank you 👍

**Pull request merged**